jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/SparkWithDataFrame.scala

```diff
@@ -14,14 +14,13 @@
  limitations under the License.
  */
 
-package ml.dmlc.xgboost4j.scala.example.spark
```

Comment:
would you mind post a new directory structure? GitHub is not convenient to review directory changes